include/af/array.h

```diff
@@ -1631,7 +1631,8 @@ extern "C" {
         @{
     */
     /**
-        \brief Gets the number of elements in an array.
+        \brief Gets the number of elements in an array. This is the product
+        of the array's dimensions.
```

Comment:
@r-barnes You forgot to change this one.

Reply:
Fixed.